src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -99,95 +99,99 @@ public void setGuiSettings(GuiSettings guiSettings) {
         }
 
         @Override
-        public Path getAddressBookFilePath() {
+        public Path getTaskListFilePath() {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public void setAddressBookFilePath(Path addressBookFilePath) {
+        public void setTaskListFilePath(Path taskListFilePath) {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public void addPerson(Person person) {
+        public void addTask(Task task) {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public void setAddressBook(ReadOnlyAddressBook newData) {
+        public void setTaskList(ReadOnlyTaskList newData) {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public ReadOnlyAddressBook getAddressBook() {
+        public ReadOnlyTaskList getTaskList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        public boolean hasTask(Task task) {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public boolean hasPerson(Person person) {
+        public void deleteTask(Task target) {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public void deletePerson(Person target) {
+        public void setTask(Task target, Task editedTask) {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public void setPerson(Person target, Person editedPerson) {
+        public void strictSetTask(Task target, Task editedTask) {
```

Comment:
Should strictSetTask be here? strictSetPerson doesn't seem to be in the original AddCommandTest.java file

Reply:
It's in the Model interface right now. I have to implement it in the model stub which implements the Model interface.

Reply:
It's in the Model interface right now. I have to implement it in the model stub which implements the Model interface.

Reply:
ohh okok!! sry about that haha